pac4j-core/src/main/java/org/pac4j/core/util/CommonHelper.java

```diff
@@ -86,6 +87,26 @@ public static boolean areNotEquals(final String s1, final String s2) {
     }
 
     /**
+     * Return if a collection is empty.
+     *
+     * @param coll a collection
+     * @return whether it is empty
+     */
+    public static boolean isEmpty(final Collection coll) {
+        return coll == null || coll.isEmpty();
+    }
+
+    /**
+     * Return if a collection is not empty.
+     *
+     * @param coll a collection
+     * @return whether it is not empty
+     */
+    public static boolean isNotEmpty(final Collection coll) {
+        return coll != null && !coll.isEmpty();
```

Comment:
To clarify, my suggestion was not so much:

```
return !coll.isEmpty()
```

But more like:

```
return !isEmpty(coll)
```

I believe this is what commons does too.